res/css/views/elements/_ImageView.scss

```diff
@@ -22,131 +22,103 @@ limitations under the License.
+.mx_ImageView_info_wrapper {
+    pointer-events: all;
+    padding-left: 32px;
     display: flex;
-    justify-content: center;
-    flex-direction: column;
-    padding-left: 30px;
-    padding-right: 30px;
-    min-height: 100%;
-    max-width: 240px;
+    flex-direction: row;
+    align-items: center;
     color: $lightbox-fg-color;
 }
 
-.mx_ImageView_cancel {
-    position: absolute;
-    // hack for mx_Dialog having a top padding of 40px
-    top: 40px;
-    right: 0px;
-    padding-top: 35px;
-    padding-right: 35px;
-    cursor: pointer;
+.mx_ImageView_info {
+    padding-left: 12px;
+    display: flex;
+    flex-direction: column;
 }
 
-.mx_ImageView_rotateClockwise {
-    position: absolute;
-    top: 40px;
-    right: 70px;
-    padding-top: 35px;
-    cursor: pointer;
+.mx_ImageView_toolbar {
+    padding-right: 16px;
+    pointer-events: all;
+    display: flex;
+    align-items: center;
 }
 
-.mx_ImageView_rotateCounterClockwise {
-    position: absolute;
-    top: 40px;
-    right: 105px;
-    padding-top: 35px;
-    cursor: pointer;
-}
+.mx_ImageView_button {
+    margin-left: 24px;
+    display: block;
 
-.mx_ImageView_name {
-    font-size: $font-18px;
-    margin-bottom: 6px;
-    word-wrap: break-word;
+    &::before {
+        content: '';
+        height: 22px;
+        width: 22px;
+        mask-repeat: no-repeat;
+        mask-size: contain;
+        mask-position: center;
+        display: block;
+        background-color: $icon-button-color;
+    }
 }
 
-.mx_ImageView_metadata {
-    font-size: $font-15px;
-    opacity: 0.5;
+.mx_ImageView_button_rotateCW::before {
+    mask-image: url('$(res)/img/image-view/rotate-cw.svg');
 }
 
-.mx_ImageView_download {
-    display: table;
-    margin-top: 24px;
-    margin-bottom: 6px;
-    border-radius: 5px;
-    background-color: $lightbox-bg-color;
-    font-size: $font-14px;
-    padding: 9px;
-    border: 1px solid $lightbox-border-color;
+.mx_ImageView_button_rotateCCW::before {
+    mask-image: url('$(res)/img/image-view/rotate-ccw.svg');
 }
 
-.mx_ImageView_size {
-    font-size: $font-11px;
+.mx_ImageView_button_zoomOut::before {
+    mask-image: url('$(res)/img/image-view/zoom-out.svg');
 }
 
-.mx_ImageView_link {
-    color: $lightbox-fg-color !important;
-    text-decoration: none !important;
+.mx_ImageView_button_zoomIn::before {
+    mask-image: url('$(res)/img/image-view/zoom-in.svg');
 }
 
-.mx_ImageView_button {
-    font-size: $font-15px;
-    opacity: 0.5;
-    margin-top: 18px;
-    cursor: pointer;
+.mx_ImageView_button_download::before {
+    mask-image: url('$(res)/img/image-view/download.svg');
 }
 
-.mx_ImageView_shim {
-    height: 30px;
+.mx_ImageView_button_more::before {
+    mask-image: url('$(res)/img/image-view/more.svg');
 }
 
-.mx_ImageView_rhs {
-    order: 3;
-    flex: 1 1 10%;
-    min-width: 300px;
-    // background-color: #800;
-    // height: 20px;
+.mx_ImageView_button_close {
+    border-radius: 100%;
+    background: #21262c;
```

Comment:
This should probably be in one of the theme files, but it's a new colour and I am not sure if an existing one shouldn't be used

Reply:
if we're hardcoding it then it's not worth it, just add a comment like `// same on all themes` to the end